### PR TITLE
Adding MEM localization method to MNE-tools

### PR DESCRIPTION
**Pull request by ilianAZZ**
This is a method to localize the superficial generators with their spatial extent using the Maximum Entropy on the Mean.

References : 
- **cMEM with stable clustering**. [MEG source localization of spatially extended generators of epileptic activity: comparing entropic and hierarchical bayesian approaches.](http://www.ncbi.nlm.nih.gov/pubmed/23418485) Chowdhury RA, Lina JM, Kobayashi E, Grova C. PLoS One. 2013;8(2):e55969

- **wMEM**. [Wavelet-based localization of oscillatory sources from magnetoencephalography data.](http://www.ncbi.nlm.nih.gov/pubmed/22410322) Lina J, Chowdhury R, Lemay E, Kobayashi E, Grova C. IEEE Trans Biomed Eng. 2012 Mar 6.

### CODE REVIEW

**Review by larsoner — COMMENTED** [review]
Link looks good!

https://output.circle-artifacts.com/output/job/871ae2f6-c9c0-4710-806e-1926a1cd9708/artifacts/0/html/install/mne_tools_suite.html

Can you add your name to `docs/changes/names.inc` and add a tiny `doc/changes/12659.other.rst` mentioning this doc update? Then we can merge

**Comment by ilianAZZ** [comment]
Hi @larsoner 

Thanks for your feedback, it's done !

File: doc/changes/names.inc [hunk 1]
```diff
@@ -220,6 +220,8 @@
 
 .. _Ilias Machairas: https://github.com/JungleHippo
 
+.. _Ilian Azz: https://github.com/ilianAZZ
```
Comment (drammock) [inline doc/changes/names.inc:223]: Ilian < Ilias alphabetically

File: doc/changes/devel/12659.other.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Adding MEM localization method to MNE-tools. by `Ilian Azz`_.
```
Comment (drammock) [inline doc/changes/devel/12659.other.rst:1]:
```suggestion
Add link to BEst-Python in the list of related software. by `Ilian Azz`_.
```

**Comment by ilianAZZ** [comment]
Done!

**Review by drammock — APPROVED** [review]

**Pull request merged**